### ISSUE TITLE
Synced logins for newly created FxA are not pulled from desktop to mobile device (iOS or Android)

### ISSUE THREAD

**Issue by SimonBasca** [mozilla/fxa-content-server#6226]
Device: iPhone X (11.2.6), Samsung S 8+ (8.0.0)

Steps to reproduce:
1. On Desktop, create a new FxA
2. Create some logins
3. Sync on desktop 
4. Login to FxA on Android or iOS

Actual Results: 
- Only one login is synced from desktop to mobile device, the saved login from the FxA.

Expected Results:
- All logins should be synced from desktop to mobile devices.

**Comment by rfk** [comment]
From #sync on slack, it sounds like this may be the same as (or very similar to) https://bugzilla.mozilla.org/show_bug.cgi?id=1461255

@SimonBasca do any of the devices involved here have a master password enabled?

**Comment by rfk** [comment]
>  Only one login is synced from desktop to mobile device, the saved login from the FxA.

To clarify, by this do you mean "the password for accounts.firefox.com, which I saved on desktop firefox when creating the account"?

**Comment by rfk** [comment]
I was able to reproduce this on my first try, using a fresh nightly profile on Desktop and a fresh build of firefox-ios on the simulator.  I saved the password for accounts.firefox.com, then did a junk login on reddit.com and news.ycombinator.com, saving both passwords.

From server data and about:sync logs, it looks like desktop failed to upload the password entries for reddit.com and news.ycombinator.com.

I think we'll need to migrate this to be a desktop Firefox bug over in bugzilla.
